src/components/funFacts/FunFactComponent.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { useState } from "react";
+import { funFacts,  FunFact  as FunFactInterface } from "./funfacts";
+import "./FunFactPage.scss";
+
+interface FunFactProps {
+    funFacts: FunFactInterface[];
+}
+
+export const FunFact: React.FunctionComponent<FunFactProps> = ({ funFacts }) => {
+
+  const [selectedFactId, setSelectedFactId] = useState(0);
+
+  const toggleFactDetail = (funFactId: number) => {
+    if (selectedFactId === funFactId){
+      setSelectedFactId(0);
+    }
+    else {
```

Comment:
Prefer `else` on same line:
```suggestion
    } else {
```